.github/workflows/test.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         uses: actions/checkout@v5
 
       - name: Setup Ruby
-        uses: ruby/setup-ruby@ab177d40ee5483edb974554986f56b33477e21d0
+        uses: ruby/setup-ruby@d5126b9b3579e429dd52e51e68624dda2e05be25
```

Comment:
This is indeed the hash for 1.267.0: https://github.com/ruby/setup-ruby/releases/tag/v1.267.0 👍